chapters/classes.tex

```diff
@@ -351,13 +351,14 @@ \subsection{Acyclic Bindings of Constants and Parameters}\label{acyclic-bindings
 
 \subsection{Component Variability Prefixes}\label{component-variability-prefixes-discrete-parameter-constant}
 
-The prefixes \lstinline!discrete!, \lstinline!parameter!, \lstinline!constant! of a component declaration are called \firstuse[variability!prefix]{variability prefixes}\index{component variability}\index{declared variability}\index{variability!declared|see{declared variability}} and define in which situation the variable values of a component are initialized (see \cref{events-and-synchronization} and \cref{initialization-initial-equation-and-initial-algorithm}) and when they are changed in transient analysis (= solution of initial value problem of the hybrid DAE):
+The prefixes \lstinline!discrete!, \lstinline!parameter!, \lstinline!constant! of a component declaration are called \firstuse[variability!prefix]{variability prefixes}\index{component variability}\index{declared variability}\index{variability!declared|see{declared variability}} and define in which situation the variable values of a component are initialized (see \cref{events-and-synchronization} and \cref{initialization-initial-equation-and-initial-algorithm}) and when they are changed during simulation:
 \begin{itemize}
 \item
-  A variable \lstinline!vc! declared with \lstinline!constant!\indexinline{constant} prefix remains constant during transient analysis, with a value that is unaffected by the initialization problem.
+  A variable \lstinline!vc! declared with \lstinline!constant!\indexinline{constant} prefix remains constant during simulation, with a value that is unaffected even by the initialization problem.
   This is called a \firstuse[---]{constant}, or \firstuse[constant!variable]{constant variable}\index{component variability!constant}.
 \item
-  A variable \lstinline!vc! declared with the \lstinline!parameter!\indexinline{parameter} prefix remains constant during transient analysis, with a value determined by the initialization problem.
+  A variable \lstinline!vc! declared with the \lstinline!parameter!\indexinline{parameter} prefix has a value determined by the initialization problem, and keeps that value during the entire simulation.
```

Comment:
We could reduce use of the lesser defined _initialization problem_ in favor of the defined _initialization_:
```suggestion
  A variable \lstinline!vc! declared with the \lstinline!parameter!\indexinline{parameter} prefix has a value determined at initialization, and keeps that value during the entire simulation.
```